src/data.jl

```diff
@@ -323,7 +323,7 @@ struct KPointGrid{D} <: AbstractKPoints{D}
     orig::SVector{D,Float64}
     function KPointGrid{D}(grid::AbstractMatrix{<:Integer}, orig::AbstractVector{<:Real}) where D
         # only allow positive values in the grid matrix
-        @assert all(x -> x > 0, grid) "negative values are disallowed in the grid matrix"
+        @assert all(x -> x >= 0, grid) "negative values are disallowed in the grid matrix"
```

Comment:
Good catch - though I actually suspect that this assertion may not be needed. I might remove this in the future.